app/src/main/java/com/example/netflixclone/MainActivity.kt

```diff
@@ -239,83 +252,88 @@ fun Greeting() {
-
-            Row(  modifier = Modifier
-                .fillMaxWidth(),
-                horizontalArrangement = Arrangement.SpaceAround)
-            {
-                Column() {
-                    Image(
-                        painter = painterResource(id = R.drawable.home),
-                        contentDescription = "home",
-                        modifier = Modifier
-                            .width(50.dp)
-                            .height(50.dp)
-                            .padding(top = 22.dp)
-                    )
-                    Text(
-                        modifier = Modifier
-                            .padding(start = 8.dp, top =  6.dp),
-                        text = "Home",
-                        color = Color.White,
-                        fontSize = 13.sp
-                    )
-                }
-                Column() {
-                    Image(
-                        painter = painterResource(id = R.drawable.news),
-                        contentDescription = "new",
-                        modifier = Modifier
-                            .width(50.dp)
-                            .height(50.dp)
-                            .padding(start = 10.dp, top = 22.dp)
-                    )
-                    Text(
-                        modifier = Modifier
-                            .padding(start = 2.dp, top =  6.dp),
-                        text = "New & Hot",
-                        color = Color.Gray,
-                        fontSize = 13.sp
-                    )
-                }
-                Column() {
-                    Image(
-                        painter = painterResource(id = R.drawable.video_game),
-                        contentDescription = "games",
-                        modifier = Modifier
-                            .width(70.dp)
-                            .height(50.dp)
-                            .padding(top = 22.dp)
-                    )
-                    Text(
-                        modifier = Modifier
-                            .padding(start = 16.dp, top =  6.dp),
-                        text = "Games",
-                        color = Color.Gray,
-                        fontSize = 13.sp
-                    )
-                }
-                Column() {
-                    Image(
-                        painter = painterResource(id = R.drawable.download),
-                        contentDescription = "downloads",
-                        modifier = Modifier
-                            .width(70.dp)
-                            .height(50.dp)
-                            .padding(top = 22.dp)
-                    )
-                    Text(
-                        modifier = Modifier
-                            .padding(start = 10.dp, top =  6.dp),
-                        text = "Downloads",
-                        color = Color.Gray,
-                        fontSize = 13.sp
-                    )
-                }
+        }
```

Comment:
Our new bottom bar that is now on same level as the Main Content column. It has no weight and will take up as much space as it needs.